packages/docker-engine/daemon-json

```diff
@@ -6,9 +6,9 @@
   "exec-opts": ["native.cgroupdriver=cgroupfs"],
   "data-root": "/var/lib/docker",
   "default-runtime": "shimpei",
-  "runtimes": { "shimpei": { "path": "shimpei" } },
+  "runtimes": { "shimpei": { "path": "shimpei", "base-runtime-spec": "/etc/docker/base-spec.json"} },
```

Comment:
I think for our case, it is OK to just specify the `daemon` configuration instead of setting it for each runtime.